### PR TITLE
[service.subtitles.subtitles.gr] 1.2.0

### PR DESCRIPTION
**Pull request by lambda81**
Updated version of the subtitles.gr service

### CODE REVIEW

File: service.subtitles.subtitles.gr/resources/lib/subztvgr.py [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+
+    def download(self, path, url):
+        try:
+            result = client.request(url)
+
+            f = os.path.splitext(urlparse.urlparse(url).path)[1][1:]
+            f = os.path.join(path, 'file.%s' % f)
+
+            file = open(f, 'wb') ; file.write(result) ; file.close()
```
Comment (phil65) [inline service.subtitles.subtitles.gr/resources/lib/subztvgr.py]: use with statement, please.

**Comment by phil65** [comment]
Please add a solid background for icon.png.

File: service.subtitles.subtitles.gr/resources/lib/subztvgr.py (hunk 1 above)
Comment (lambda81) [inline service.subtitles.subtitles.gr/resources/lib/subztvgr.py]: Added it in my new PR
https://github.com/lambda81/service.subtitles.subtitles.gr/blob/master/resources/lib/subztvgr.py#L141
https://github.com/lambda81/service.subtitles.subtitles.gr/blob/master/resources/lib/subtitlesgr.py#L134
https://github.com/lambda81/service.subtitles.subtitles.gr/blob/master/resources/lib/xsubstv.py#L143

**Comment by lambda81** [comment]
Added it in my new PR
https://github.com/lambda81/service.subtitles.subtitles.gr/blob/master/icon.png

**Pull request closed without merging**